lldb/source/Plugins/LanguageRuntime/Swift/SwiftMetadataCache.cpp

```diff
@@ -77,6 +87,7 @@ void SwiftMetadataCache::registerModuleWithReflectionInfoID(ModuleSP module,
     return;
   }
 
+  auto UUID = module->GetUUID().GetBytes();
```

Comment:
It might be easier to make cachedUUID a StringRef and us llvm::toStringRef() on the result of GetBytes() here?

Reply:
What's the advantage of that over having both as `ArrayRefs`?

Reply:
in that case my comment doesn't make a whole lot of sense
I was just trying to minimize the amount of casting.